pkg/executables/kubectl.go

```diff
@@ -578,7 +583,7 @@ func (k *Kubectl) RemoveAnnotationInNamespace(ctx context.Context, resourceType,
 }
 
 func (k *Kubectl) GetEksaCluster(ctx context.Context, cluster *types.Cluster) (*v1alpha1.Cluster, error) {
-	params := []string{"get", eksaClustersResourceType, cluster.Name, "-o", "json", "--kubeconfig", cluster.KubeconfigFile}
+	params := []string{"get", "clusters", "-A", "-o", "jsonpath={.items[0]}", "--kubeconfig", cluster.KubeconfigFile, "--field-selector=metadata.name=" + cluster.Name}
```

Comment:
If you do this, you can ensure that it will always pick the right cluster and then compare namespace values from that? What happens if there are multiple clusters with the same name in different namespaces (which is an edge case)? We may want a check for that to error out?